docker-parse/README.md

```diff
@@ -0,0 +1,11 @@
+# Run the Open Source Parse Server in Docker on Microsoft Azure
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fdocker-parse%2Fazuredeploy.json" target="_blank">
+	<img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fdocker-parse%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template allows you to deploy an Ubuntu Server 15.04 VM with Docker (using the [Docker Extension](https://github.com/Azure/azure-docker-extension))
```

Comment:
Use the 15.10 image. 15.04 is EOL and will be removed from the image repository soon causing this template to break